### ISSUE TITLE
Rebuild image for Rust 1.52.1

### ISSUE THREAD

**Issue by gibfahn** [emk/rust-musl-builder#122]
**Rust release:** 1.52.1 (stable)

Have wanted to use [`String::split_once()`](https://doc.rust-lang.org/std/string/struct.String.html#method.split_once) a couple of times now.

**Comment by JMLX42** [comment]
I am also interested in this.

**Comment by altanozlu** [comment]
any news 🤔

**Comment by gibfahn** [comment]
Closing in favour of https://github.com/emk/rust-musl-builder/issues/123

**Comment by emk** [comment]
This is blocked on: https://github.com/emk/rust-musl-builder/pull/126. Please see the discussion there.

**Comment by emk** [comment]
We now have up-to-date nightly builds using GitHub CI! Thank you to everyone for the encouragement.
